README.md

```diff
@@ -1,107 +1,149 @@
 # Overview
```

Comment:
To my experience, people rarely go far away from the README, and, usually, they get confused by hierarchical documents with lots of links, that's why I decided to put most of the stuff in the README. Another consideration is that I want the motivational examples to be just at the start of the page, otherwise people might just run away without clicking any links at all. This is the reason, why I put the python example here.